README.md

```diff
@@ -22,11 +22,11 @@
 </p>
 
 <p align="center">
-  <a href="https://wxt.dev/guide/installation.html" target="_blank">Get Started</a>
+  <a href="https://wxt.dev/guide/installation" target="_blank">Get Started</a>
   &bull;
-  <a href="https://wxt.dev/api/config.html" target="_blank">Configuration</a>
+  <a href="https://wxt.dev/api/config" target="_blank">Configuration</a>
```

Comment:
I don't think removing all HTML suffixes is meaningful. It breaks my redirects I have setup for old links too.

https://github.com/wxt-dev/wxt/blob/main/docs/public/_redirects

Reply:
Look, in my opinion there should be a single standard - no suffix, because shorter links look better, about those links with a suffix - maybe it's good, but I don't see any point in it...

Reply:
I mean that if you went to a page with a suffix, you will go to a page with a suffix, if without a suffix, then without a suffix

I just want a single link type without a suffix to be used in the future

Reply:
I mean that if you went to a page with a suffix, you will go to a page with a suffix, if without a suffix, then without a suffix

I just want a single link type without a suffix to be used in the future

Reply:
I agree, I'd prefer no suffix, but the current netlify and vitepress setup defaults to .html for everything. So that's what we should continue to use for now 🤷